packages/aws-cdk-lib/core/lib/private/unique-resource-name.ts

```diff
@@ -46,7 +46,7 @@ const MAX_LEN = 256;
 
 const HASH_LEN = 8;
 
-export function makeUniqueResourceName(components: string[], options: MakeUniqueResourceNameOptions) {
+export function makeUniqueResourceName(components: string[], options: MakeUniqueResourceNameOptions, prefix: string='') {
```

Comment:
Let's add this as an option instead independently like this.

Reply:
Done!